### PR TITLE
add deadletterservice and jobdeskintegrationservice

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: dev/app-data.yml [hunk 1]
```diff
@@ -120,6 +120,46 @@ services:
             placement:
                 constraints: [node.labels.type == data]
 
+    jobdeskintegrationservice-postgres:
+        image: postgres:latest
```
Comment (ivanmrva) [inline dev/app-data.yml]: don't use latest, but the version as used for all the other postgres images

File: dev/app-data.yml [hunk 2]
```diff
@@ -120,6 +120,46 @@ services:
...
+            - jobroom_data2
+        secrets:
+            - postgres-passwd
+        command: postgres -c tcp_keepalives_idle=60 -c tcp_keepalives_interval=60 -c tcp_keepalives_count=60
+        deploy:
+            placement:
+                constraints: [node.labels.type == data]
+
+    deadletterservice-postgres:
+        image: postgres:latest
```
Comment (ivanmrva) [inline dev/app-data.yml]: same here

File: dev/app.yml [hunk 3]
```diff
@@ -117,6 +117,40 @@ services:
...
+            SERVER_PORT: 8090
+        ports:
+            - 8090:8090
+        networks:
+            - jobroom_data2
+        secrets:
+            -   source: bootstrap.yml-1
+                target: bootstrap.yml
+        deploy:
+            placement:
```
Comment (ivanmrva) [inline dev/app.yml]: placement "java" is not used anymore, pls remove

File: dev/app.yml [hunk 4]
```diff
@@ -117,6 +117,40 @@ services:
...
+            SERVER_PORT: 8091
+        ports:
+            - 8091:8091
+        networks:
+            - jobroom_data2
+        secrets:
+            -   source: bootstrap.yml-1
+                target: bootstrap.yml
+        deploy:
+            placement:
```
Comment (ivanmrva) [inline dev/app.yml]: same here

File: dev/app-data.yml (hunk 1 above)
Comment (fabiangotzen) [reply dev/app-data.yml]: done

File: dev/app.yml (hunk 3 above)
Comment (fabiangotzen) [reply dev/app.yml]: removed

**Pull request merged**